bve-build/src/main.rs

```diff
@@ -142,70 +121,41 @@ fn build(options: &Options) {
     assert!(child.wait().expect("Unable to wait for cargo.").success());
 }
 
-fn handle_cbindgen_error(err: &cbindgen::Error, options: &Options) {
-    if let cbindgen::Error::CargoExpand(s, err) = &err {
-        // Bug in cbindgen/rustc
-        // https://github.com/eqrion/cbindgen/issues/457
-        // https://github.com/rust-lang/rust/issues/68333
-        // Fixed by cleaning the build cache and rerunning
-        if err.to_string().contains("Finished") && s == "bve-native" {
-            println!("Dealing with cbindgen bug; clearing cache and regenerating");
-            clean();
-            generate_c_bindings(options);
-            return;
-        }
-    }
-    panic!("cbindgen error: {}", err)
-}
-
-fn generate_c_bindings(options: &Options) {
-    let config = cbindgen::Config::from_file("bve-native/cbindgen.toml").unwrap();
-
-    {
-        // C
-        let mut config = config.clone();
-        config.language = Language::C;
-        *config.header.as_mut().expect("bve-native/cbindgen.toml needs a header") +=
-            "/* C API for BVE-Reborn high performance libraries. */";
-        let result = cbindgen::Builder::new()
-            .with_crate("bve-native")
-            .with_config(config)
-            .generate();
-        match result {
-            Ok(bindings) => {
-                bindings.write_to_file("bve-native/include/bve.h");
-            }
-            Err(err) => handle_cbindgen_error(&err, options),
-        }
-    }
-    {
-        // C++
-        let mut config = config;
-        config.language = Language::Cxx;
-        config.export.prefix = None;
-        *config.header.as_mut().expect("bve-native/cbindgen.toml needs a header") +=
-            "/* C++ API for BVE-Reborn high performance libraries. */";
-        config.trailer = Some(read_to_string("bve-native/include/bve_cpp.hpp").unwrap());
-        let result = cbindgen::Builder::new()
-            .with_crate("bve-native")
-            .with_config(config)
-            .generate();
-        match result {
-            Ok(bindings) => {
-                bindings.write_to_file("bve-native/include/bve.hpp");
-            }
-            Err(err) => handle_cbindgen_error(&err, options),
-        }
-    }
+fn generate_c_bindings() {
+    println!("Generating C Bindings... (may take a while)");
+    let mut c = Command::new("cbindgen")
```

Comment:
You're using command-line cbindgen instead of compiling it yourself? Sounds dope

Reply:
Yup, this is how I get <2s build times